SpatialGDK/Documentation/content/get-started/shooter-tutorial.md

```diff
@@ -0,0 +1,3 @@
+# Get started: 4 - Multi-Server Shooter Tutorial
```

Comment:
We need to standardize on multserver or multi-sever as we are going between the two.

Reply:
Also "tutorial" lower case initial.